### PR TITLE
ISSUE-3 Added Dockerfile for Debian based image

### PR DESCRIPTION
**Pull request by juanmatias**
## what
* Create Debian based Toolbox image
* Replace old Alpine Dockerfile with new Debian one

## why
* To avoid some issues related to Alpine

## references
* closes https://github.com/binbashar/le-docker-leverage-toolbox/issues/3
* closes https://github.com/binbashar/le-docker-leverage-toolbox/issues/39

Note on the Issue 39. Since the Debian based image will be the default one (no Alpine will exist), the `-debian` suffix itps not needed anymore. Thus, from this ticket only the automation is needed, discarding the naming considerations.

### CODE REVIEW

**Comment by juanmatias** [comment]
The scope of this PR is fulfilled. New tickets will be created for further activities, please @diego-ojeda-binbash @exequielrafaela , could you approve this one?

File: Dockerfile [hunk 1]
```diff
@@ -1,19 +1,34 @@
...
+
+FROM debian:11.6-slim AS base
+
+LABEL vendor="Binbash Leverage (info@binbash.com.ar)"
+
+################################
+# Versions
+################################
+
+ARG GLIBC_VERSION
```
Comment (diego-ojeda-binbash) [inline Dockerfile:19]: @juanmatias Are we really using this?

File: Dockerfile [hunk 2]
```diff
@@ -22,41 +37,94 @@ RUN set -ex && \
...
+ARG AWSCLI_VERSION
+ARG AWSVAULT_VERSION
+ARG HCLEDIT_VERSION
+
+################################
+# Install 
+################################
+
 # Add aws-vault
 RUN curl -LO "https://github.com/99designs/aws-vault/releases/download/${AWSVAULT_VERSION}/aws-vault-linux-amd64" \
```
Comment (diego-ojeda-binbash) [inline Dockerfile:109]: @juanmatias @angelofenoglio @exequielrafaela Can't remember when we started using this? Do you recall?

**Review by diego-ojeda-binbash — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (diego-ojeda-binbash) [reply Dockerfile:109]: Yeah, I don't think it's used anywhere. Let's remove it and see what breaks (since rolling back should be pretty easy).

Comment (juanmatias) [reply Dockerfile]: done

**Pull request merged**